lib/LedgerSMB/Report/Listings/TemplateTrans.pm

```diff
@@ -110,18 +110,14 @@ sub name {
 
 =cut
 
-my %jtype = (
-    1 => 'gl',
-    2 => 'ar',
-    3 => 'ap',
-    );
+my @jtype = ( undef, 'gl', 'ar', 'ap' );    # XXX magic number map.  From where is the magic? --rir
```

Comment:
My guess is that it equates to journal types in the database, but the hashed list was too short.
That seems pretty obvious to me now.  But I'd rather be overcautious than...